renderdoc/driver/d3d12/d3d12_command_list_wrap.cpp

```diff
@@ -4485,9 +4470,10 @@ bool WrappedID3D12GraphicsCommandList::Serialise_ClearDepthStencilView(
     {
       if(m_Cmd->InRerecordRange(m_Cmd->m_LastCmdListID))
       {
-        Unwrap(m_Cmd->RerecordCmdList(m_Cmd->m_LastCmdListID))
-            ->ClearDepthStencilView(Unwrap(DepthStencilView), ClearFlags, Depth, Stencil, NumRects,
-                                    pRects);
+        ID3D12GraphicsCommandListX *list = m_Cmd->RerecordCmdList(m_Cmd->m_LastCmdListID);
+
+        TRANSFER_WITH_ACTION_CALLBACK(Unwrap(list)->ClearDepthStencilView(
```

Comment:
Clears and other actions that aren't draws or dispatches should use a `Misc` type to match the vulkan callback - i.e. `PostDraw` etc for draws, `PostDispatch` etc for dispatches, and `PostMisc` for others, with action flags to indicate what type of action it is. See the definition of `VulkanActionCallback`.